docs/usage/vmware-netplan-guestinfo-metadata.md

```diff
@@ -0,0 +1,36 @@
+---
+nav_order: 2
+parent: Usage
+---
+
+# VMware Netplan guestinfo metadata
+
+The network environment can vary between VMware servers and instead of leaking these requirements into userdata snippets, a well-known guestinfo metadata field can be used.
+The guestinfo metadata field is OS-independent and supported by cloud-init and afterburn. It requires Netplan to render the Netplan YAML format to either NetworkManager or systemd-networkd configuration files.
```

Comment:
Here and below: we capitalize Afterburn in prose.

Reply:
Thanks, done